main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
I thought about that. Just checking hasAttribute and returning a default value might work for some attributes. If user wants an attribute like "style" which is meant to be an easy interface to adjusting the attribute of the element, then it seems like that would be insufficient. Really we'd want the element to be initialized with those attributes in that case, which might be a bit more involved to implement. Thoughts?